introduction.md

```diff
@@ -1 +1,4 @@
-This is an introduction. I am doing the Mastering GitHub course. Favourite Code School path is HTML & CSS
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?